virtualization/hyperv_on_windows/user_guide/vmsession.md

```diff
@@ -34,13 +34,13 @@ Enter-PSSession -VMGUID <VMGUID>
 Exit-PSSession 
 ``` 
 
-> Note:  If you're session won't connect, make sure you're using credentials for the virtual machine you're connecting to -- not the Hyper-V host.
+> Note:  If your session won't connect, make sure you're using credentials for the virtual machine you're connecting to -- not the Hyper-V host.
 
 To learn more about these cmdlets, see [Enter-PSSession](http://technet.microsoft.com/library/hh849707.aspx) and [Exit-PSSession](http://technet.microsoft.com/library/hh849743.aspx). 
 
 ## Run a script or command with Invoke-Command
 
-You can use the [Invoke-Command](http://technet.microsoft.com/library/hh849719.aspx) cmdlet to run a pre-determined set of commands on the virtual machine. Here is an example of how you can use the Invoke-Command cmdlet where PSTest is the virtual machine name and the script to run (foo.ps1) is in the script folder on the C:/ drive:
+You can use the [Invoke-Command](http://technet.microsoft.com/library/hh849719.aspx) cmdlet to run a pre-determined set of commands on the virtual machine. Here is an example of how you can use the Invoke-Command cmdlet where PSTest is the virtual machine name and the script to run (foo.ps1) is in the script folder on the C:\ drive:
```

Comment:
C:\\ is a path, not a drive. Should be one of `C: drive:`, `C drive`, or `root directory of the C drive`. And to be really pedantic :smiling_imp:, all of that's wrong as you probably mean the system drive... :smile_cat: